concepts/05 Widgets/Form/20 Change Options at Runtime/15 Editor Options.md

```diff
@@ -58,6 +62,104 @@ With Angular, bind the option to change in the [editorOptions](/api-reference/10
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div>
+            <DxForm :form-data="employee">
+                <DxSimpleItem data-field="firstName" :editor-options="{disabled: isFirstNameDisabled}" />
```

Comment:
In Vue and React, we do not recommend declaring objects in the markup. I suppose we will have bind editorOption to an object and reassign the entire object.

Reply:
Fixed https://github.com/DevExpress/devextreme-documentation/pull/628/commits/6fef4f3309f3225bf7c28b6e32ddebf89045804f